rust-toolchain.toml

```diff
@@ -0,0 +1,5 @@
+[toolchain]
+channel = "1.81.0"
+profile = "minimal"
```

Comment:
I think the profile is not needed, realistically we want "complete" or "default" on dev machines and "minimal" in CI, and if we don't specify it here, the one from the rustup setup will be used.